.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+  - "nightly"  # nightly build
```

Comment:
Which version is "nightly" right now? A few weeks ago Python 3.10.0 was released

Reply:
I thing so

Reply:
yes 3.10 https://app.travis-ci.com/github/RedHatInsights/insights-results-aggregator-utils/jobs/546635132

Reply:
added 3.9 too